deso/block.go

```diff
@@ -353,6 +355,68 @@ func (node *Node) getBlockTransactionsWithHypersync(blockHeight uint64, blockHas
 		})
 	}
 
+	for pk, balance := range stakedDESOBalances {
```

Comment:
Add a comment here explaining your schema. It looks like you have: `VALIDATOR_ENTRY:ADDRESS -> balance`, which mirrors what we do with Creator Coins. I assume this is just to keep track of the total amount staked per validator, which acts as a "pool" of funds that gets credited when someone stakes and debited when someone unstakes. Explain that dynamic here, and maybe even link to the design doc so that future generations understand why we did it in this particular way.

Reply:
We just read the bytes of the validator PKID into the core `PublicKey` struct to simplify things. I did this so we could reuse a lot of the logic for creator coin balances and regular deso balances which require a map of `map[lib.PublicKey]uint64`. If you'd prefer, I can just add the new functions and use PKID to make this more clear.